src/components/notifications/GoogleChat.vue

```diff
@@ -10,4 +10,35 @@
             </i18n-t>
         </div>
     </div>
+
+    <div class="mb-3">
+        <div class="form-check form-switch">
+            <input id="google-chat-use-template" v-model="$parent.notification.googleChatUseTemplate" type="checkbox" class="form-check-input">
+            <label for="google-chat-use-template" class="form-check-label"> {{ $t("Use plain text template instead of CardsV2") }} </label>
```

Comment:
Two things:
- please add all translation keys to `en.json`. If they are not there, our translators cannot translate them
- I am not quite sure if we want to be this technical in the message. What do you think about

```suggestion
            <label for="google-chat-use-template" class="form-check-label"> {{ $t("Template plain text instead of using cards") }} </label>
```